### PR TITLE
[hitcount] re arrange the script

### PR DESCRIPTION
**Pull request by bw1**
travis: hitcount 	✘ 	✔ 	✔ 	4 	☡ 	(c)

* move the instructions from sub hitcount to sub  refresh_hitcount
* correct initializing of the variables

### CODE REVIEW

**Comment by dequis** [comment]
Usually helps to describe what you are changing and why you're doing it.

File: scripts/hitcount.pl [hunk 1]
```diff
@@ -128,28 +141,27 @@ sub hitcount {
...
-    get_hitcount();
-    Irssi::statusbar_items_redraw('hitcount');
-}
-
 sub read_settings {
     my $time = Irssi::settings_get_int('hitcount_refresh');
     return if ($time == $last_refresh);
 
     $last_refresh = $time;
     Irssi::timeout_remove(my $refresh_tag) if (my $refresh_tag);
```
Comment (ailin-nemui) [inline scripts/hitcount.pl]: here is an error, this refresh_tag should be a global variable

**Comment by ailin-nemui** [comment]
You still need to remove the tag every time otherwise you get many many timers running in parallel

**Comment by ailin-nemui** [comment]
can you please use four space indentation in your changed lines, to keep in style with the rest of the script? also, with `UNLOAD` you went a bit too excessive. Timeouts are cleared on unload automatically :-)

**Comment by bw1** [comment]
@ailin-nemui

**Comment by ailin-nemui** [comment]
you need to rebase this because now it conflicts in  _testing/config.yml  due to your hddtemp fix before I can merge it

**Pull request merged**